cmake/autogenerated_versions.txt

```diff
@@ -2,7 +2,7 @@
 
 # NOTE: has nothing common with DBMS_TCP_PROTOCOL_VERSION,
 # only DBMS_TCP_PROTOCOL_VERSION should be incremented on protocol changes.
-SET(VERSION_REVISION 2)
+SET(VERSION_REVISION 3)
```

Comment:
A mintor version update or a major one

Reply:
it's not necessary, the version will increases automatically after release.
This is only to ensure that CI testing can handle the status based on the new revision.